feature/schedule/src/main/java/com/yapp/feature/schedule/ScheduleScreen.kt

```diff
@@ -138,7 +138,9 @@ private fun ScheduleAllScreen(
     onIntent: (ScheduleIntent) -> Unit
 ) {
     LazyColumn(
-        modifier = Modifier.fillMaxSize(),
+        modifier = Modifier.fillMaxSize()
+            .navigationBarsPadding()
+        ,
```

Comment:
```suggestion
        modifier = Modifier.fillMaxSize(),
```
위에 코멘트는 제가 착각했네요... 무시해주시면 됩니다!

<img width="579" alt="image" src="https://github.com/user-attachments/assets/002c10f2-5c73-4473-b97b-a921eb215117" />

진욱님이 말씀하신대로 YappBackground에서 WindowInsets.systemBars를 기반으로 전체 safe area padding을 추가하고 있어서 navigationBarsPadding()은 빼도 무방한 것 같습니다

Reply:
@DongChyeon 오 확인 감사합니다 .. !!! 제가 나중에 공통 컴포넌트나 컨벤션에 대한 걸 리드미로 작성해 놓을게요 ..

Scaffold의 innerPadding이 적용되어서 BottomNavBar Component의 Height는 자동으로 무시되어야할 것 같은데 좀 이상하긴 하네요... 혹시 fillMaxSize 말구 weight(1f)로 한번 바꿔봐주실 수 있나용? @ashwon12

Reply:
앗 그렇게 해서 해봤었어요 !! 똑같았습니닷
fillMaxSize 제외하고 위 아래 weight (1f) 넣었었는데 똑같이 패딩값 적용이 안되더라구요